test/conftest.py

```
@@ -37,3 +43,114 @@
+    go2rtc_command = [
+        "./go2rtc",  # Adjust path to go2rtc executable if needed
+        "--config",
+        str(config_path),
+    ]
+    process = subprocess.Popen(go2rtc_command)
+    print(process.returncode)
+    print(process.stderr)
+    print(process.stdout)
+    try:
+
+        def is_rtsp_stream_ready(rtsp_address):
```

Ruff (0.11.9)

Missing return type annotation for private function `is_rtsp_stream_ready`

Add return type annotation: `bool | None`

(ANN202)